### PR TITLE
Permissions endpoint can now be filtered, sorted and paginated

### PR DESCRIPTION
**Pull request by leplatrem**
Ref #600 #653 

- [x] Add documentation.
- [x] Add tests.
- [x] Add a changelog entry.
- [x] Find a way to allow GET as anonymous `#needhelp`
- [ ] Approach may be a bit too hacky (c.f `last_modified` missing from perm entries)

feedback? @Natim

### CODE REVIEW

**Comment by Natim** [comment]
> Find a way to allow GET as anonymous

I think this is when you have a permission dynamic set on it and that you match on the ``system.Everyone`` principal.

**Comment by Natim** [comment]
I am wondering if we should go that way and not just add filtering and sorting to the previous service.

**Comment by Natim** [comment]
After investigation it seems that you are using a `UserResource` rather than a `ShareableResource` Data from a UserResource are linked to a user so you cannot link data to a non authenticated user.

If we use a ShareableResource it should work.

**Comment by Natim** [comment]
Ok I manage to make it works.

**Comment by Natim** [comment]
What would it actually takes to add the `last_modified` value? Do we need it?

File: kinto/core/errors.py [hunk 1]
```diff
@@ -171,6 +171,7 @@ def raise_invalid(request, location='body', name=None, description=None,
     :raises: :class:`~pyramid:pyramid.httpexceptions.HTTPBadRequest`
     """
     request.errors.add(location, name, description, **kwargs)
+    request.errors.request = request  # Needed by json_error_handler to reapply_cors
```
Comment (leplatrem) [inline kinto/core/errors.py]: https://github.com/mozilla-services/cornice/issues/378

**Pull request merged**